lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -401,6 +412,13 @@ type ListByronTransactions n = "byron-wallets"
     :> QueryParam "order" (ApiT SortOrder)
     :> Get '[JSON] [ApiTransaction n]
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/postByronTransactionFee
+type PostByronTransactionFee n = "byron-wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "payment-fees"
```

Comment:
cannot be `txs-fees`, `transactions-fees` or just `fees`?

Reply:
This is to distinguish between delegation-fees and payment-fees. Both are transactions, but they are different kind of transactions.